bindings/pyroot/cppyy/CPyCppyy/src/CPPOverload.cxx

```diff
@@ -144,6 +145,41 @@ static inline PyObject* HandleReturn(
     int ll_action = 0;
     if (result) {
 
+     // If the "smart pointer" is only an observing pointer that indicated
```

Comment:
Can you add to the commit a patch file that contains the changes done to cppyy? Especifically here: https://github.com/root-project/root/tree/master/bindings/pyroot/cppyy/patches